adserver/templates/adserver/reports/staff-regiontopics.html

```diff
@@ -21,17 +21,17 @@
   <label class="col-form-label" for="id_region">{% trans 'Region' %}</label>
   <select class="form-control" name="region" id="id_region">
     <option value="">All Regions</option>
-    {% for name in region_list %}
-      <option value="{{ name }}"{% if region == name %} selected{% endif %}>{{ name }}</option>
+    {% for reg in region_list %}
+      <option value="{{ reg.slug }}"{% if region == reg.slug %} selected{% endif %}>{{ reg.name }}</option>
     {% endfor %}
   </select>
 </div>
 <div class="col-xl-3 col-md-6 col-12 mb-3">
   <label class="col-form-label" for="id_topic">{% trans 'Topic' %}</label>
   <select class="form-control" name="topic" id="id_topic">
     <option value="">All Topics</option>
-    {% for name in topic_list %}
-      <option value="{{ name }}"{% if topic == name %} selected{% endif %}>{{ name }}</option>
+    {% for tp in topic_list %}
```

Comment:
It's probably nicer to use `region` and `topic` here, unless we're worried about it clobbering that variable from somewhere else?

Reply:
That is exactly what I'm avoiding. `region` and `topic` are set in the context.